smart-contracts/contracts/CosmosBridge.sol

```diff
@@ -117,37 +117,40 @@ contract CosmosBridge is CosmosBridgeStorage, Oracle {
         address signer,
         bytes32 hashDigest,
         uint8 _v,
-		bytes32 _r,
-		bytes32 _s
+		    bytes32 _r,
```

Comment:
These whitespace changes actually did not happen. I'm not sure why they're showing here.